tests/ui/filter_methods.stderr

```diff
@@ -1,3 +1,12 @@
+error: non-binding `let` on a type that implements `Drop`
```

Comment:
> Or better, ...

As an outsider, a change like that looks to me like it would go against an established convention. Maybe it would be better left for a separate PR?

Reply:
It is an ancient convention to warn on all lints in test files. Now one test file _should_ only test one lint. But we don't care enough about this to fix that.

So both's fine.